ajax/libs/goalProgress/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "goalProgress",
+  "filename": "goalProgress.min.js",
+  "version": "1.0",
+  "description": "Animated progress bar using jQuery to show how close you are to reaching your goal.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinacious/goalProgress.git"
+  },
+  "keywords": [
+    "goalProgress",
+    "tags",
+    "jQuery",
+    "animated"
+  ],
+  "author": "tinacious",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tinacious/goalProgress.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
```

Comment:
The basePath should be `""`
Because we get the file from root dir not from `dist` folder.
Thank you.